rest-api-guidelines/execution/long-running-tasks/README.md

```diff
@@ -0,0 +1,10 @@
+# Long Running Tasks
+
+This section includes the recommended approaches to handling long runnint tasks (LRTs) in REST APIs. 
```

Comment:
@jesusjavierdediego  typo  `runnint`

Reply:
Fixed typos.
Changes in the cache section.